supplementary/alica_ros1/alica_tests/src/test/test_config_change.cpp

```diff
@@ -42,8 +42,7 @@ TEST_F(AlicaNotInitialized, TestUpdatingComponents_002)
 
 TEST_F(AlicaNotInitialized, TestUpdatingComponents_003)
 {
-    EXPECT_TRUE(ac->setOption<bool>("Alica.AllowIdling", false));
-    EXPECT_FALSE(PartialAssignment::isIdlingAllowed());
+    PartialAssignment::allowIdling(false);
```

Comment:
why did this change?

Reply:
what was before is in 2 lines below now. This was rebased, so it had changed naturally in the system from an older version to a new version.

Reply:
I checked here and with the addition of this back it was failing tests, I fixed it.

Reply:
`EXPECT_FALSE(PartialAssignment::isIdlingAllowed());` was wrong. I don't know much about what this is testing, but is this supposed to be TRUE?